lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1593,7 +1593,11 @@ def plot_surface(self, X, Y, Z, *args, norm=None, vmin=None,
             Bounds for the normalization.
 
         shade : bool
-            Whether to shade the face colors.
+            Whether to shade the facecolors. Defaults to True. Shading is
+            always disabled when `cmap` is specified.
+
+        lightsource : `~matplotlib.colors.LightSource`
+            The lightsource to use when `shade` is True.
```

Comment:
We use `*shade*` to denote other variables.

https://matplotlib.org/devel/documenting_mpl.html#id3

Reply:
Looks like I missed this instance, sorry.